Cargo.toml

```diff
@@ -11,10 +11,13 @@ repository = "https://github.com/myfreeweb/systemstat"
 
 [dependencies]
 time = "0.1"
-chrono = "0.4"
+chrono = {version = "0.4.19", features = ["serde"] }
```

Comment:
Missing space after the `{`. Any reason to bump the version?

Reply:
Again sorry my vscode extension did that automatically. I missed the space